tests/test_securityhub/test_securityhub.py

```diff
@@ -154,3 +157,423 @@ def test_get_findings_max_results():
     assert isinstance(get_response["Findings"], list)
     assert len(get_response["Findings"]) == 1
     assert "NextToken" in get_response
+
+
+@mock_aws
+def test_enable_organization_admin_account():
+    org_client = boto3.client("organizations", region_name="us-east-1")
+    org_client.create_organization(FeatureSet="ALL")
+
+    admin_account_id = "123456789012"
+    org_client.create_account(
+        AccountName="SecurityHubAdmin",
+        Email="securityhub.admin@example.com",
+    )
+
+    accounts = org_client.list_accounts()["Accounts"]
+    admin_account = next(acc for acc in accounts if acc["Name"] == "SecurityHubAdmin")
+    admin_account_id = admin_account["Id"]
+
+    client = boto3.client("securityhub", region_name="us-east-1")
+    client.enable_organization_admin_account(AdminAccountId=admin_account_id)
+
+    get_resp = client.get_administrator_account()
+    assert "Administrator" not in get_resp
+
+    with mock.patch.dict(os.environ, {"MOTO_ACCOUNT_ID": admin_account_id}):
```

Comment:
```suggestion
    if not settings.TEST_DECORATOR_MODE:
        # Can't set env variables in ServerMode
        return
    
    with mock.patch.dict(os.environ, {"MOTO_ACCOUNT_ID": admin_account_id}):
```
We will have to skip these parts in ServerMode. Moto runs in a separate process there, so updating environment variables does not work 

Exitting early allows us to keep the rest of the test intact, so we still test the 'regular' (non-multi account) parts.

Note that settings should be a direct import from moto:
```
from moto import settings
```

This applies to all the other tests where we set environment variables as well, of course.